plugins/RocketsPlugin/SnapshotTask.h

```diff
@@ -77,7 +77,6 @@ class SnapshotFunctor : public TaskFunctor
             *_camera = *_params.camera;
         else
             *_camera = engine.getCamera();
-        _camera->setAspectRatio(float(_params.size.x()) / _params.size.y());
```

Comment:
you sure? the snapshot has a different ratio. it's handled in the assignment now?